bitbots_ball_filter/cfg/BallFilter.cfg

```diff
@@ -28,7 +29,6 @@ group_ROS.add("ball_filter_reset_service_name", str_t, 0, "Name of service for r
 
 
 group_filter.add("filter_rate", int_t, 0, "Filtering rate in Hz", min=0, max=255)
-group_filter.add("min_ball_confidence", double_t, 0, "Minimum ball confidence", min=0.0, max=1.0)
```

Comment:
Why do we not want to filter by confidence?

Reply:
I really don't like that this is in here.

Reply:
I really do like it, because the approach taken in this pr dosnt work other wise, because of the return in case of a low conficene.

Reply:
Also the ball confidence is thresholded 3(!) times in the pipeline with different values. The 2 in the vision have a reason, but this one here is totally unessesary imo.

Reply:
AH! then we should meve that to a later point or handle the situation differently. However, I truly believe that the ball filter should be the component selecting the used ball measurements and not the vision or the transformer.

Reply:
Lets talk in person about this. I am tried of writing stuff ^^

Reply:
What was the result of this discussion?